plugin/v2/plugin.go

```diff
@@ -309,7 +358,7 @@ func (p *Plugin) InitSpec(s specs.Spec, libRoot string) (*specs.Spec, error) {
 			m.Source = *mount.Source
 		}
 		if m.Source != "" && m.Type == "bind" {
-			fi, err := os.Lstat(filepath.Join(rootfs, m.Destination)) // TODO: followsymlinks
+			fi, err := os.Lstat(filepath.Join(p.Rootfs, string(os.PathSeparator), m.Destination)) // TODO: followsymlinks
```

Comment:
@crosbymichael I think this code was trying to make sure the source directory is created.